cmd/client/root.go

```diff
@@ -31,4 +32,59 @@ func register(app *grumble.App) {
 		},
 	})
 
+	app.AddCommand(&grumble.Command{
+		Name: "strlen",
+		Help: "str value length",
+		Run:  stringStrLen,
+		Args: func(a *grumble.Args) {
+			a.String("key", "key", grumble.Default(""))
+		},
+	})
+
+	app.AddCommand(&grumble.Command{
+		Name: "getset",
+		Help: "get curvalue set newvalue",
```

Comment:
The content of `Help` needs to be modified to indicate what data structure this is.  As shown in the figure below, you need to indicate which data structure to add or get data into.

The same below.

![image](https://github.com/ByteStorage/FlyDB/assets/90094807/4849a5c3-d89e-4c20-af48-1c0a2f1ceb0e)